protocol/messenger_messages.go

```diff
@@ -287,27 +287,29 @@ func (m *Messenger) DeleteMessageForMeAndSync(ctx context.Context, chatID string
 	response.AddChat(chat)
 
 	if m.hasPairedDevices() {
-		clock, _ := chat.NextClockAndTimestamp(m.getTimesource())
-
-		deletedForMeMessage := &DeleteForMeMessage{}
+		err = m.withChatClock(func(chatID string, clock uint64) error {
+			deletedForMeMessage := &protobuf.DeleteForMeMessage{
+				MessageId: messageID,
+				Clock:     clock,
+				ChatId:    message.ChatId,
```

Comment:
Not sure I understand why this fixes the issue, isn't the chat id already specified in `common.RawMessage` (line 304)?

Reply:
line 304 won't be transfered to protocol.DeleteForMeMessage.LocalChatID

Reply:
Sorry, I still don't see how it fixes the "sync message for delete for me should not be sent to someone else", as to whom the message is sent is declared in `common.RawMesssage` or am I missing something?

Reply:
> Do I get it right?
you are right 😉